aws-ec2-transitgatewayroutetable/resource-role.yaml

```diff
@@ -0,0 +1,41 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: >
+  This CloudFormation template creates a role assumed by CloudFormation
+  during CRUDL operations to mutate resources on behalf of the customer.
+
+Resources:
+  ExecutionRole:
+    Type: AWS::IAM::Role
+    Properties:
+      MaxSessionDuration: 8400
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: resources.cloudformation.amazonaws.com
+            Action: sts:AssumeRole
+            Condition:
+              StringEquals:
+                aws:SourceAccount:
+                  Ref: AWS::AccountId
+              StringLike:
+                aws:SourceArn:
+                  Fn::Sub: arn:${AWS::Partition}:cloudformation:${AWS::Region}:${AWS::AccountId}:type/resource/AWS-EC2-TransitGatewayRouteTable/*
+      Path: "/"
+      Policies:
+        - PolicyName: ResourceTypePolicy
+          PolicyDocument:
+            Version: '2012-10-17'
+            Statement:
+              - Effect: Allow
+                Action:
+                - "ec2:CreateTags"
+                - "ec2:CreateTransitGatewayRouteTable"
+                - "ec2:DeleteTransitGatewayRouteTable"
+                - "ec2:DescribeTransitGatewayRouteTables"
```

Comment:
This was auto generated I have not modified this file

Reply:
Did you submit the resource manually and test? I think it will need ec2:DescribeTags, ec2:DeleteTags . Does the delete tag operation work without this?